apps/examples/stdk_smart_lamp/Makefile

```diff
@@ -167,6 +138,32 @@ $(BUILTIN_REGISTRY)$(DELIM)$(FUNCNAME).bdat: $(DEPCONFIG) Makefile
 	$(Q) $(call REGISTER,$(APPNAME),$(FUNCNAME),$(THREADEXEC),$(PRIORITY),$(STACKSIZE))
 
 context: $(BUILTIN_REGISTRY)$(DELIM)$(FUNCNAME).bdat
+####################################
```

Comment:
To be commands in a tab, a line should be started with a tab.
https://www.gnu.org/software/make/manual/html_node/Rule-Syntax.html#Rule-Syntax
```
The recipe lines start with a tab character
```

Is this called when context is executed?